flake.nix

```diff
@@ -35,6 +35,7 @@
                         (old: {
                           executableSystemDepends = (old.executableSystemDepends or []) ++ [
                             final.boost.dev
+                            final.nixVersions.nix_2_24
```

Comment:
I am very unsure about this line, since I don't know the haskell nix infrastructure.
It feels like their must be a better way to override this.

Reply:
Looks good to me, having a reasonable amount of experience maintaining a few packages over the years.